skgstat/Variogram.py

```diff
@@ -2383,6 +2384,39 @@ def nrmse(self):
         """
         return self.rmse / np.nanmean(self.experimental)
 
+    @property
+    def root_mean_square(self):
+        """Root Mean Square (RMS) of the residuals
+
+        Calculates the square root of the mean of squared residuals.
+
+        Returns
+        -------
+        float
+            Root Mean Square of the residuals.
+        """
+        return np.sqrt(np.nanmean(np.square(self.model_residuals)))
+
+    @property
+    def residual_sum_of_squares(self):
+        """Residual Sum of Squares (RSS)
+
+        Calculates the sum of squared differences between the experimental
+        variogram and theoretical model values.
+
+        Returns
+        -------
+        float
+            Residual sum of squares (RSS), a measure of the overall model fit
+            representing the sum of squared deviations between the observed
+            experimental variogram and the corresponding theoretical model values.
+        """
+        return np.nansum(np.square(self.model_residuals))
+
+    @property
+    def rss(self):
+        return self.residual_sum_of_squares
```

Comment:
I added this, cause I like to have both, the full name and the short name.

Reply:
Sounds great